plugins/Files/js/components/filelist.js

```diff
@@ -26,36 +24,40 @@ const FileList = ({files, selected, searchResults, path, showSearchField, action
 		filelistFiles = files
 	}
 	const fileElements = filelistFiles.map((file, key) => {
-		if (file.type === 'directory') {
-			return (
-				<Directory key={key} onClick={onDirectoryClick(file)} name={file.name} />
-			)
-		}
-		const isSelected = selected.includes(file.siapath)
+		const isSelected = selected.map((selectedfile) => selectedfile.name).includes(file.name)
```

Comment:
I think `selected.some((selectedFile) => selectedFile.name === file.name)` is equivalent, though arguably less readable (`some` isn't a great name for this purpose)